src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -147,6 +147,10 @@ class: "Symfony\Component\VarDumper\Tests\Caster\ReflectionCasterTest"
      */
     public function testGenerator()
     {
+        if (extension_loaded('xdebug')) {
```

Comment:
should be skipped only when ``xdebug.overload_var_dump`` is set to 2:

- 0 does not overload var_dump at all
- 1 overloads it only with formatting, which does not break our tests AFAIK
- 2 adds the location to the formatted output.